stp.py

```diff
@@ -40,446 +38,494 @@
 rootW = -1
 rootH = -1
 
+
 # Classes
 class Edge:
-	classType = "Edge"
```

Comment:
Please change it back to tabs, so I can see properly what changes you made :)

Reply:
That would be a very tedious process, as my code uses spaces by default. I've tried to revert 9d6fe268f7588686b16e282ab8ca5b5ea8070cd7 for you. But, that just gives indentation errors and conflicts. To see my actual changes you can look at the last 5 commits. Github review gives you the option to look at a certain commit range. 

The 9d6fe268f7588686b16e282ab8ca5b5ea8070cd7 commit changes no functionality it just auto reformats the file based on Pycharms code styles.

Reply:
I've manually inspected the commits and I'll accept the changes - PR looks fine. I can live with the change from tabs to spaces.